opm/grid/CpGrid.hpp

```diff
@@ -937,7 +937,7 @@ namespace Dune
 
 
         const std::vector<double>& zcornData() const {
-            return data_->zcornData();
+            return data_[0]->zcornData();
```

Comment:
I think you discovered a bug here. This should probably have operated on current_view_data_ .  should probably check whether this function is used.

Reply:
It seems to be only used in `createEclipseGrid` in `GridHelpers.cpp`.